src/components/ui/text-field.tsx

```diff
@@ -240,7 +241,7 @@ const styles = StyleSheet.create({
   },
   placeholder: {
     position: 'absolute',
-    color: PLACEHOLDER_GRAY,
```

Comment:
If there are no colors in the Color enum, then the constants should be left

Reply:
done